user-access-handlers/src/main/java/no/unit/nva/handlers/data/DefaultRoleSource.java

```diff
@@ -81,13 +72,15 @@ public class DefaultRoleSource implements RoleSource {
 
     private static final RoleDto CURATOR_THESIS_ROLE = RoleDto.newBuilder()
                                                            .withRoleName(CURATOR_THESIS_ROLE_NAME)
-                                                           .withAccessRights(List.of(MANAGE_DEGREE))
+                                                           .withAccessRights(List.of(MANAGE_RESOURCES_STANDARD,
```

Comment:
Since Nvi,Doi,support,publishing-curators have MANAGE_RESOURCES_STANDARD i assumed we also should have it for the last 2 curator-types @ketilaa

Reply:
I am not sure that is the intention. I will check it with Jan Erik.